emission/net/ext_service/geocoder/nominatim.py

```diff
@@ -94,6 +99,82 @@ def reverse_geocode(cls, lat, lng):
             return _do_google_reverse(lat, lng) # Just in case
 
 ## Failsafe section
+def check_against_business_location(location='0, 0', address = ''):
```

Comment:
wait, why are you passing in the location as a string instead of two ints? converting to/from string seems to be a lot of extra work.

Reply:
This is still unresolved....

Reply:
and it also just looks ugly. using a comma separated set of ints is a presentation issue, conforming to the format for query parameters (I think). You should format values for presentation as late as possible to give greater flexibility in case of changes.